game-app/game-headless/src/main/java/org/triplea/game/server/debug/ChatAppender.java

```diff
@@ -2,37 +2,28 @@
 
 import ch.qos.logback.classic.spi.ILoggingEvent;
 import ch.qos.logback.core.AppenderBase;
+import com.google.common.base.Preconditions;
+import games.strategy.engine.chat.Chat;
 import java.util.List;
-import java.util.Optional;
-import org.triplea.game.server.HeadlessGameServer;
 
 /**
  * A {@link AppenderBase} that publishes log records to the chat subsystem. This allows a headless
  * game server to report its own logs to other game clients via the chat window.
  */
 public final class ChatAppender extends AppenderBase<ILoggingEvent> {
-  private boolean enabled = true;
+  private final Chat chat;
 
-  @Override
-  protected void append(final ILoggingEvent record) {
-    // guard against infinite recursion if sendChatMessage also logs
-    if (enabled) {
```

Comment:
For reference:
This guard is removed because logback's `AppenderBase` class already does this